### PR TITLE
Clean up the package, update package() to respect the AUR etiquette

### PR DESCRIPTION
**Pull request by closingin**
Hello!

Here is a PR to do some clean up on this AUR package, and fix the problem with the wrong path being added when executing the `self:shell-setup` command. I also removed the manual extraction of the archive as makepkg can do it on its own :slightly_smiling_face: 

Actually, setting up the cli goes against the package etiquette from the AUR as per this text that you can find [here](https://wiki.archlinux.org/index.php/Arch_package_guidelines#Package_etiquette):

> All important messages should be echoed during install using an .install file. For example, if a package needs extra setup to work, directions should be included.

So I switched to a message displayed after the installation to inform the user about the command! You can see it in the `symfony-cli.install` file.

Feel free to tell me if you want to change anything.

### CODE REVIEW

File: PKGBUILD [hunk 1]
```diff
@@ -1,25 +1,18 @@
...
 pkgname=symfony-cli
-pkgrel=1
+pkgrel=2
 pkgver=4.11.3
 pkgdesc="The Symfony client helps developers create and manage Symfony applications."
 url="https://symfony.com/"
 arch=('x86_64')
-license=('custom')
-install=''
+license=('MIT')
```
Comment (famoser) [inline PKGBUILD]: Is the license here concerning the AUR package definition (which indeed is MIT) or concerning the package itself (which I do not know the license of)? The package itself is proprietary (https://symfony.com/legal/cli-eula).

File: PKGBUILD [hunk 2]
```diff
@@ -1,25 +1,18 @@
...
+install="${pkgname}.install"
 source=(
     "https://github.com/symfony/cli/releases/download/v${pkgver}/symfony_linux_amd64.gz"
 )
-noextract=('symfony_linux_amd64.gz')
```
Comment (famoser) [inline PKGBUILD:14]: makes much more sense if arch extracts it directly, thanks :)

File: PKGBUILD (hunk 1 above)
Comment (closingin) [reply PKGBUILD]: Good point! I just checked and it seems it's the licensing of the software itself, so yeah it should remain `custom`. Gonna change it.

However, from most of the proprietary software I've checked on the AUR (see how [spotify](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=spotify#n92) does it), it seems that your `LICENSE` file should be the license of the software, to then copy it in the `/usr/share/licenses` directory. Are you fine if I replace it?

Comment (famoser) [reply PKGBUILD]: The spotify link is handy, thanks!

Not sure however for this package this the cleanest approach, as the AUR package definition by itself does not fall under the symfony CLI license.

I would instead create a new file for this (`Symfony CLI End-User License Agreement`) and copy this into the correct folder with `install -Dm 644 "Symfony CLI End-User License Agreement" "${pkgdir}/usr/share/licenses/${pkgname}/Symfony CLI End-User License Agreement"`. 

Additionally, I would place a notice in the `post_install()` that the user "accepts the terms and conditions of this Agreement by using the Application" or something along these lines. 

I can also do this by myself and would now merge your changes; your choice.

Comment (closingin) [reply PKGBUILD]: Fixed! :slightly_smiling_face:

**Comment by famoser** [comment]
thanks for the help, just published the new version :)

**Pull request merged**